frontend/language/src/en.json

```diff
@@ -2,8 +2,12 @@
   "dashboard": "Dashboard",
   "dashboard.delete.form": "Delete form",
   "dashboard.download": "Download data",
+  "dashboard.empty.message": "Press the button on the right to create your first form!",
+  "dashboard.forms.below": "Your forms are shown below.",
```

Comment:
Here also i think it is additional information that maybe we do not need. 😊

Reply:
Agreed 👍